prebid-video/video-overview.md

```diff
@@ -81,6 +83,32 @@ The ad server chooses the winning line item. If a Prebid line item is selected,
 5.	**Prebid renders the outstream video.**
 Prebid retrieves the winning outstream video bid and renderer from the ad ID (hb_adid).  Prebid renders the outstream video into the banner slot.
 
+### Long-form Video 
+
+1.	**Prebid sends request to demand partners.**
+Prebid.js code loads within the page header and sends a bid request to each video demand partner included on a given Prebid video ad unit.
+
+2.	**Demand partners respond.**
```

Comment:
Just a point of feedback (could be left alone).  

When rendering this page, the formatting of the text with the emboldened sentence changes after the first two steps.  The text of the first two steps are in-line with the bold sentence, while the text for the rest of the steps have a line-break.  

See the screenshot below for what I mean.
![inline_vs_linebreak](https://user-images.githubusercontent.com/31102355/53178908-bd740100-35c0-11e9-8737-97bbdeafbd37.png)

Reply:
Fixed